src/web/js/trove/chart-lib.js

```diff
@@ -1556,7 +1576,7 @@
             .append(redrawG);
         }
 
-        if(!hasImage) { return; } // If we don't have images, our work is done!
+        if(!hasImage && !dotChartP) { return; } // If we don't have images, our work is done!
```

Comment:
...and this would become `if(!replaceDefaultSVG)`

Reply:
Done